test/Microsoft.AspNet.Mvc.Core.Test/ActionResults/HttpNotFoundObjectResultTest.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.WebUtilities;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc
+{
+    public class HttpNotFoundObjectResultTest
+    {
+        [Fact]
+        public void HttpNotFoundObjectResult_InitializesStatusCode()
+        {
+            // Arrange & act
+            var notFound = new HttpNotFoundObjectResult(null);
+
+            // Assert
+            Assert.Equal(StatusCodes.Status404NotFound, notFound.StatusCode);
```

Comment:
A better test for this would be to execute the result and verify that it sets status code on the response. Tests that very that a property is set by the constructor are of little value :laughing: